models/stg_zendesk__ticket_field_history.sql

```diff
@@ -32,13 +37,15 @@ final as (
         field_name,
         {% if target.type == 'redshift' -%}
             cast(updated as timestamp without time zone) as valid_starting_at,
```

Comment:
I know this isn't in the scope of your changes. But couldn't we just have this leverage `{{ dbt.type_timestamp() }}` instead of doing the conditional logic for Redshift? I am fairly certain this is what we do in all other packages.

Reply:
i'll check!

Reply:
according to the [code](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/adapters/include/global_project/macros/utils/data_types.sql#L16-L33) + postgres comment i think we should be okay to swap